behat.yml.dist

```diff
@@ -6,6 +6,8 @@ default:
                 - Behatch\Context\BrowserContext:
                     timeout: 3
                 - Behatch\Context\XmlContext
+                - Behatch\Context\DebugContext:
+                    screenshotDir: "var/cache/selenium/"
```

Comment:
It's more `var/logs` than `var/cache` no ?
Not sure it should be enabled all the time. I would prefer to activate it on demand per developer thanks to `behat.yml` file instead of having it all the time. What do you think ?